src/polyswarm_api/types/resources.py

```diff
@@ -107,18 +107,25 @@ def __init__(self, json, polyswarm=None):
         metadata = {metadata['tool']: metadata['tool_metadata'] for metadata in metadata_json}
         self.metadata = Metadata(metadata, polyswarm)
 
-        self._detections = None
+        self._malicious = None
```

Comment:
These fields should be coming back from the api now, no need to use properties, we just need to parse the json.

Reply:
Could you be more specific? I know that I get a missing attribute error in the formatter if I don't create these methods/properties.